recipe/conda_build_config.yaml

```diff
@@ -14,29 +14,37 @@ xxhash:
 
 # Coupling dependencies specification together.
 # The first  line comes from the specifications of the `libevent-2.1.10` branch.
-# The second line comes from the alternative specifications of the `libevent-2.1.10` branch.
-# The third  line comes from the specifications of the `main` branch.
+# The second and third line come from the alternative specifications of the `libevent-2.1.10` branch.
+# The fourth line comes from the specifications of the `main` branch.
 # See: https://docs.conda.io/projects/conda-build/en/stable/resources/variants.html#coupling-keys
+# "*" means that the dependency is chosen implicitly to be the one of
+# conda-forge global pinning:
+# See: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml
```

Comment:
This is not what `"*"` means, unfortunately; what this does is _override_ the global pinning with "any version goes" constraint, and so you will always pull in the newest version, even if conda-forge hasn't migrated for this version yet.

Reply:
Thank you for pointing this out.

I forgot that the versions on the global pinning were not necessarily the last ones. We wanted to have the latest versions for the AWS SDK, hence only the comment need to be changed.

Reply:
Corrected with 817366cfef031f32a3a95148cf5d52b1a1b473bc (a `[ci skip]` commit pushed to `rc` directly).